src/aead/aes.rs

```diff
@@ -266,7 +266,7 @@ impl Key {
                 ctr32_encrypt_blocks!(vpaes_ctr32_encrypt_blocks, in_out, src, &self.inner, ctr)
             }
 
-            #[cfg(any(target_arch = "x86"))]
```

Comment:
Please rebase on top of the latest main. to get rid of the need for this change.

Reply:
Rebasing done (71dc9e9 and 04282ec). See what changed [here](https://github.com/briansmith/ring/compare/05e5107560048a683dd2521b61c9273eb005ead5..04282ec737556361fdd724ac08b490df1b5c84ad).